docs/samples/python/Presidio_with_Flair_as_a_3rd_party_recognizer

```diff
@@ -105,42 +151,38 @@ class FlairRecognizer(EntityRecognizer):
         :param explanation: Explanation string
         :return:
         """
+        
         explanation = AnalysisExplanation(
             recognizer=FlairRecognizer.__class__.__name__,
             original_score=original_score,
-            textual_explanation=f"Identified as {entity_type} by Flair Recognizer",
+            textual_explanation=explanation,
         )
 
         return explanation
+    
+    @staticmethod
+    def __check_label(
+        entity: str, label: str, check_label_groups: Tuple[Set, Set]
+    ) -> bool:
+        return any(
+            [entity in egrp and label in lgrp for egrp, lgrp in check_label_groups]
+        )
 
-
-# Define Presidio Analyzer
-analyzer = AnalyzerEngine()
-
-# Define a text to identify entities
-text = 'Antoni Gaudi is the most famous architect in Barcelona, La Sagrada Familia is not finished yet, but you can enjoy its stone magic forest.'
-
-# Analyze text with Presidio.
-results = analyzer.analyze(text, language='en')
-for entity in results:
-    print(f'type: {entity.entity_type},\t start: {entity.start}, \t end: {entity.end},\
-          score: {entity.score},\t entity: {text[entity.start:entity.end]}')
-
-# type: PERSON,	 start: 0, 	 end: 12,          score: 0.85,	 entity: Antoni Gaudi
-# type: LOCATION,	 start: 45, 	 end: 54,          score: 0.85,	 entity: Barcelona
-
-# Include our Flair recognizer to the analyzer
-flair_recognizer = FlairRecognizer()
-analyzer.registry.add_recognizer(flair_recognizer)
-
-# Analyze text with Presidio and our Flair analyzer.
-results = analyzer.analyze(text, language='en')
-for entity in results:
-    print(f'type: {entity.entity_type},\t start: {entity.start}, \t end: {entity.end},\
-          score: {entity.score},\t entity: {text[entity.start:entity.end]}')
-
-# type: PER,	 start: 0, 	 end: 12,          score: 1.0,	 entity: Antoni Gaudi
-# type: LOC,	 start: 45, 	 end: 54,          score: 0.99,	 entity: Barcelona
-# type: PERSON,	 start: 0, 	 end: 12,          score: 0.85,	 entity: Antoni Gaudi
-# type: LOCATION,	 start: 45, 	 end: 54,          score: 0.85,	 entity: Barcelona
-# type: ORG,	 start: 56, 	 end: 74,          score: 0.76,	 entity: La Sagrada Familia
+    
+if __name__ == '__main__':
+    
+    # Illustrative example only: Run Presidio analyzer
```

Comment:
This is great! I also tried running it as part of the AnalyzerEngine, here's the code I used, feel free to include if you feel it's useful:

```python
from presidio_analyzer import AnalyzerEngine, RecognizerRegistry


flair_recognizer = FlairRecognizer() # This would download a very large (+2GB) model on the first run

registry = RecognizerRegistry()
registry.add_recognizer(flair_recognizer)

analyzer = AnalyzerEngine(registry=registry)

results = analyzer.analyze("My name is Christopher and I live in Irbid", language="en", return_decision_process=True)
for result in results:
    print(result)
    print(result.analysis_explanation)
```

For this sentence, the Flair model detects Location ("Irbid") but spaCy doesn't :)

Reply:
Perfect, I will modify the code with this example.